<?xml version="1.0" encoding="utf-8"?>
<!-- This URDF was automatically created by SolidWorks to URDF Exporter! Originally created by Stephen Brawner (brawner@gmail.com) 
     Commit Version: 1.6.0-4-g7f85cfe  Build Version: 1.6.7995.38578
     For more information, please see http://wiki.ros.org/sw_urdf_exporter -->
<robot
  name="r2Arm">
  <link
    name="base_link">
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://r2_arm_test_01/meshes/base_link.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="1 1 1 1" />
      </material>
    </visual>
  </link>
  <link
    name="Link1">
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://r2_arm_test_01/meshes/Link1.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="1 1 1 1" />
      </material>
    </visual>
  </link>
  <joint
    name="joint1"
    type="revolute">
    <origin
      xyz="0 0.18031 -0.0183"
      rpy="-1.5708 1.5708 0" />
    <parent
      link="base_link" />
    <child
      link="Link1" />
    <axis
      xyz="1 0 0" />
      <limit effort="1000.0" lower="0" 
    upper="6.28" velocity="0.5"/>
  </joint>
  <link
    name="Link2">
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://r2_arm_test_01/meshes/Link2.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="1 1 1 1" />
      </material>
    </visual>
  </link>
  <joint
    name="joint2"
    type="continuous">
    <origin
      xyz="0.159 0 -0.324"
      rpy="0.020886 0 -3.1416" />
    <parent
      link="Link1" />
    <child
      link="Link2" />
    <axis
      xyz="-1 0 0" />
  </joint>
  <link
    name="Link3">
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://r2_arm_test_01/meshes/Link3.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="1 1 1 1" />
      </material>
    </visual>
  </link>
  <joint
    name="joint3"
    type="revolute">
    <origin
      xyz="-0.024 0 -0.075"
      rpy="0 -0.36369 0" />
    <parent
      link="Link2" />
    <child
      link="Link3" />
    <axis
      xyz="0 -1 0" />
    <limit effort="1000.0" lower="0.0" 
    upper="1.57" velocity="0.5"/>

  </joint>
  
  
</robot>